recipe/build.sh

```diff
@@ -41,7 +41,7 @@ if [[ "${cuda_compiler_version:-None}" != "None" ]]; then
     elif [[ ${cuda_compiler_version} == 11.1 ]]; then
         export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
     elif [[ ${cuda_compiler_version} == 11.2 ]]; then
-        export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
+        export TF_CUDA_COMPUTE_CAPABILITIES=sm_52,sm_60,sm_70,sm_80,compute_90
```

Comment:
Pretty sure 11.2 doesn't support `90`, see e.g. [here](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-feature-list) or [here](https://github.com/conda-forge/magma-feedstock/blob/main/recipe/build.sh) or [here](https://gitlab.kitware.com/cmake/cmake/-/blob/v3.27.7/Modules/CUDA/architectures.cmake?ref_type=tags#L50)

Also, AFAIU, we shouldn't remove 75/86. The reason for this has never been particularly self-evident to me, so perhaps someone who might know better feels like chiming in?
CC @adibbley @leofang @carterbox @jakirkham

Reply:
Already fixed locally with the values from the recent Tensorflow PR

Reply:
You can prune the minor archs if compilation time is a concern. The embedded PTXs would be jit'd at run time.

Reply:
Nice, I will probably do this as otherwise CI will timeout.

Reply:
Yah, remember to embed the PTXs (compute_XX), I didn't check what's expected for setting `TF_CUDA_COMPUTE_CAPABILITIES`, though, speaking solely from NVCC perspective